### PR TITLE
fix "Enable using a custom npm registry" issue

### LINKED ISSUE #35: Enable using a custom npm registry
The npm resolver does not take the registry in the .bowerrc into account. Is it possible to install artifacts from a different npm registry as https://registry.npmjs.org and how?

### PR DESCRIPTION
**Pull request by madbonez**
Fixes #35
Get some code from here https://github.com/npm/npm/blob/latest/lib/config/pacote.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bower-npm-resolver",
-  "version": "0.9.0",
+  "version": "0.9.1",
```
Comment (mjeanroy) [inline package.json]: The version number will be updated automatically once released, can you revert this line?

**Comment by mjeanroy** [comment]
@madbonez Thank you! Can you revert the version number update please (once done, I'll publish a new release)?

**Comment by madbonez** [comment]
Done

**Comment by cheplv** [comment]
@mjeanroy When you think new release can be done?

**Pull request merged**